src/YHttpWidgetsActionHandler.cc

```diff
@@ -473,22 +474,11 @@ int YHttpWidgetsActionHandler::do_action(YWidget *widget, const std::string &act
         }
         else if( dynamic_cast<YMenuButton*>(widget) )
         {
-            return action_handler<YMenuButton>( widget, body, [&] (YMenuButton *mb) {
-                // Vector of string to store path to the tree item
-                std::vector<std::string> path;
-                boost::split( path, value, boost::is_any_of( TreePathDelimiter ) );
-                YMenuItem * item = mb->findItem( path );
-                if ( item )
-                {
-                    yuiMilestone() << "Activating Item by path :" << value << " in \"" << mb->label() << "\" MenuButton" << std::endl;
-                    mb->setKeyboardFocus();
-                    activate_widget( mb, item );
-                }
-                else
-                {
-                    throw YUIException("Item with path: '" + value + "' cannot be found in the MenuButton widget");
-                }
-            } );
+            return get_menu_selector_handler( dynamic_cast<YMenuButton*>(widget), value, body);
```

Comment:
Just coding style, missing space after `body`.

Reply:
good catch! I was opening with space and closing with it.
What I see is that for method is consistent to use those spaces but for methods not, and there are other things. I applied ctrl+shift+I in my vscode and you can check the result in an additional commit I added. I think it is much more clear, but we can think of another tool more common, I haven't check exactly what vscode applys. wdyt?

Reply:
The problem here is that there is no standard tool to apply this coding style, so it happened to me multiple times that I have missed something. So the only chance is code review. But that one of the ideas where we collaborate and apply some solution to whole libyui project.

Reply:
it would be great, if there would be some agreement, I noticed I missed another one :) it would save time in review plus the code would look nicer.